rllib/core/learner/learner.py

```diff
@@ -1441,6 +1440,16 @@ def after_gradient_based_update(self, *, timesteps: Dict[str, Any]) -> None:
                 `NUM_ENV_STEPS_SAMPLED_LIFETIME`.
                 # TODO (sven): Make this a more formal structure with its own type.
         """
+        # Log total loss accross all modules.
```

Comment:
I guess the single loss terms are still stored in the `(mid, "total_loss")` keys, correct? I think its important to see which module contributes how much to the total loss.